modules/internal/ChapelTuple.chpl

```diff
@@ -17,10 +17,26 @@
  * limitations under the License.
  */
 
-// ChapelTuple.chpl
-//
-// tuple data implementation as a record
-//
+/*
+Predefined Functions on Tuples.
+
+Tuples are a predefined structured type in Chapel. They are specified
+in the Tuples chapter of the Chapel Language Specification.
+This page lists the predefined functions on tuples.
+They are always available to all Chapel programs.
+
+Besides the functions defined here, the Chapel Language specification
+defines other operations available on tuples: indexing, iteration,
+assignment, and unary, binary, and relational operators.
+
+The following method is also availble:
+
+  .. code-block:: chapel
+
+    proc tuple.size param
```

Comment:
Out of curiosity, do you know where the implementation of size lives? Is it in some other module, or in the compiler?